hangman.py

```diff
@@ -0,0 +1,173 @@
+            urllib.request.urlretrieve(url, filePath)
+            print('file downloaded to ' + filePath)
+        finally:
+            # storing the file contents as a list
+            with open(filePath) as f:
+                self.lines = f.readlines()
+            print('file read')
+
+    def wordPick(self):
+        # This method randomly selects a word of a minimum length from the stored list
+        # self.minlength = int(args.minlength)
+        self.word = str()  # word chosen by game
+
+        # selecting a random word from the list that has a length greater than the specified minimum
+        while len(self.word) < self.minlength:
+            self.word = random.choice(self.lines).lower()
+            self.word = self.word[:-1]
+
+    def printhangman(self, wrongGuesses):
+        # This method prints on of the 7 states of the hangman
+        # This method prints on of the 7 states of the hangman
+        print('  |‾‾‾‾‾‾‾|')  # printing top line of gallows for all states
+
+        if wrongGuesses > 0:  # printing second line of gallows for all states
+            print('  |       O')
+        else:
+            print('  |')
+
+        if wrongGuesses == 6:  # printing third line of gallows for all states
+            print('  |      /|\\')
+        elif wrongGuesses == 5:
+            print('  |      /|')
+        elif wrongGuesses >= 2:
+            print('  |       |')
+        else:
+            print('  |')
+
+        if wrongGuesses >= 4:  # printing fourth line of gallows for all states
+            print('  |      / \\')
+        elif wrongGuesses == 3:
+            print('  |      / ')
+        else:
+            print('  |')
+
+        print('__|____\n')  # printing fifth line of gallows for all states
+
+    def newGame(self):
+
+        self.openFile()
+        self.wordPick()
+
+        letters = list(string.ascii_lowercase)
+        guessed = copy.deepcopy(letters)
+        guesses = ''
+        target = ''
+        output = ''
+        stateHM = 0
+
+        for char in self.word:
+            target += ' ' + char + ' '
+
+        for char in self.word:
+            output += ' _ '
+        print(output)
+
+        while stateHM < 6:
+            output = ''
+
+            print(*guessed, sep=' ')
+            guess = input("Choose a letter: ")
+            guess = guess.lower()
+
+            if guess in letters:
+                if guess in guessed:
+                    guessed.remove(guess)
+
+                if guess not in guesses:
+                    guesses += guess
+
+                    if guess not in self.word:
+                        print(f'{guess} is not in the word!')
+                        stateHM += 1
+                    else:
+                        print(f'{guess} is in the word!')
+
+                else:
+                    print(f'You already guessed "{guess}"!')
+
+                time.sleep(1)
+                self.printhangman(stateHM)
```

Comment:
similar to the camel case comment, lets try to use "_" to demarcate words in our functions for ease of reading